PracticeGit/conversion.c

```diff
@@ -0,0 +1,41 @@
+// ----------------------------------------------------------------------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ----------------------------------------------------------------------------------------------------------------------------------------------------------------
+// conversion.c
+// Program on A3 branch.
+// ----------------------------------------------------------------------------------------------------------------------------------------------------------------
+#include "conversion.h"
+
+int binary[BITS];   // Global array to hold binary representation
+
+void DecimalToBinary (int n) {
+   for (int i = 0; i < BITS; i++)  binary[i] = (n >> i) & 1;
+}
+
+void FormatBinary (char formattedBinary[BITS / 8 * 9 + 1]) {

```

Comment:
```suggestion
void FormatBinary (char formattedBinary[]) {
```
See [here](https://stackoverflow.com/questions/15542697/c-function-argument-char-vs-char#:~:text=All%20strings%20are%20pointers%20to%20arrays%20of%20char%2C,just%20another%20way%20of%20writing%20the%20same%20thing.)